{
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "psirt@us.ibm.com",
            "DATE_PUBLIC": "2021-05-28T00:00:00",
            "ID": "CVE-2019-4722",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "Cognos Analytics",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": "11.0"
                                            },
                                            {
                                                "version_value": "11.1"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "IBM"
                    }
                ]
            }
        },
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "IBM Cognos Analytics 11.0 and 11.1 could allow a remote attacker to obtain sensitive information via a stack trace due to mishandling of certain error conditions. IBM X-Force ID: 172128."
                }
            ]
        },
        "impact": {
            "cvssv3": {
                "BM": {
                    "A": "N",
                    "AC": "L",
                    "AV": "N",
                    "C": "L",
                    "I": "N",
                    "PR": "L",
                    "S": "U",
                    "SCORE": "4.300",
                    "UI": "N"
                },
                "TM": {
                    "E": "U",
                    "RC": "C",
                    "RL": "O"
                }
            }
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "Obtain Information"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://www.ibm.com/support/pages/node/6451705",
                    "refsource": "CONFIRM",
                    "title": "IBM Security Bulletin 6451705 (Cognos Analytics)",
                    "url": "https://www.ibm.com/support/pages/node/6451705"
                },
                {
                    "name": "ibm-cognos-cve20194722-info-disc (172128)",
                    "refsource": "XF",
                    "title": "X-Force Vulnerability Report",
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/172128"
                }
            ]
        }
    },
    "NVD": {
        "configurations": {
            "CVE_data_version": "4.0",
            "nodes": []
        },
        "cve": {
            "CVE_data_meta": {
                "ASSIGNER": "psirt@us.ibm.com",
                "ID": "CVE-2019-4722"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "en",
                        "value": "IBM Cognos Analytics 11.0 and 11.1 could allow a remote attacker to obtain sensitive information via a stack trace due to mishandling of certain error conditions. IBM X-Force ID: 172128."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": []
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://www.ibm.com/support/pages/node/6451705",
                        "refsource": "CONFIRM",
                        "tags": [],
                        "url": "https://www.ibm.com/support/pages/node/6451705"
                    },
                    {
                        "name": "ibm-cognos-cve20194722-info-disc (172128)",
                        "refsource": "XF",
                        "tags": [],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/172128"
                    }
                ]
            }
        },
        "impact": {},
        "lastModifiedDate": "2021-06-01T14:33Z",
        "publishedDate": "2021-06-01T14:15Z"
    }
}